app/models/user_option.rb

```diff
@@ -14,6 +14,7 @@ class UserOption < ActiveRecord::Base
   scope :human_users, -> { where('user_id > 0') }
 
   enum default_calendar: { none_selected: 0, ics: 1, google: 2 }, _scopes: false
+  enum sidebar_topic_destination: { latest: 0, unread: 1 }, _scopes: false
```

Comment:
"latest" is not quite right here because the default view of a category is actually configurable. See https://github.com/discourse/discourse/blob/7718ef97cc8a87589f2246d9f1ec139a3a53682e/app/models/category.rb#L1062